app/api/v2/entities/activity.rb

```diff
@@ -3,7 +3,7 @@
 module API::V2
   module Entities
     class Activity < Grape::Entity
-      format_with(:iso_timestamp) { |d| d.utc.iso8601 }
+      format_with(:iso_timestamp) { |d| d&.utc&.iso8601 }
```

Comment:
so what will be format is d is nil ?

Reply:
it will be `nil`
But as soon, as a param is `nil` you dont need to format it.
It will just skip.

We already have the `d&.utc&.iso8601` in user entity and one more, and it works perfect